fips202/native/aarch64/profiles/cortex_a55.h

```diff
@@ -15,7 +15,7 @@
 // On Cortex-A55, we use lazy rotation assembly for Keccak-x1,
 // but no batched assembly implementation.
 #define MLKEM_USE_FIPS202_X1_NATIVE
-static inline void keccak_f1600_x1_native(uint64_t *state)
+static INLINE void keccak_f1600_x1_native(uint64_t *state)
```

Comment:
I wonder if instead we should have `STATIC_INLINE`? We already have `STATIC_INLINE_TESTABLE`

Reply:
hmm. Can you elaborate why you think that is better?

Reply:
I find `STATIC_INLINE` + `STATIC_INLINE_TESTABLE` more uniform than `STATIC_INLINE_TESTABLE + static INLINE`. But this just taste, let's leave it.